src/main/groovy/org/pih/warehouse/api/StockMovementItem.groovy

```diff
@@ -131,6 +133,8 @@ class StockMovementItem {
                 quantityCanceled          : quantityCanceled,
                 quantityRevised           : quantityRevised,
                 quantityPicked            : quantityPicked,
+                unitOfMeasure             : unitOfMeasure,
+                packsRequested            : quantityRequested / packSize,
```

Comment:
are you sure we don't have to do any additional checks in terms of dividing those two values, like checking is `packSize` is always `> 0`?